src/components/LineChart.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { Line } from 'react-chartjs-2';
+import { Col, Row, Typography } from 'antd';
+
+const { Title } = Typography;
+
+const LineChart = ({ coinHistory, currentPrice, coinName }) => {
+    const coinPrice = [];
+    const coinTimestamp = [];
+
+    for(let i = 0; i < coinHistory?.data?.history?.length; i += 1) {
```

Comment:
Nice, there are many ways of iterating through a list.  For loops are just the tip of the iceberg.